app/views/divisions/history.html.haml

```diff
@@ -2,14 +2,16 @@
   = Diffy::CSS
 
 - content_for :title do
-  Division Description Changes —
-  = @division.original_name
-  —
-  = formatted_date(@division.date)
+  Division Description Changes
 - set_meta_tags description: "All changes made to the description and title of the division: #{@division.name}"
 
 .page-header
-  %h1= yield :title
+  %h1
+    = yield :title
+  %h2
+    = @division.original_name
+    %small.pre-title
```

Comment:
Should the date be nested in the h2? I'm really not sure about this - we should check and make it consistent with the rest of the app (I'm guessing we use this pattern elsewhere?).